tensorrt_llm/_torch/auto_deploy/utils/_config.py

```
@@ -41,7 +41,8 @@ def __init__(self, *args, **kwargs):
                 "specify the `yaml_default` field in your pydantic model instead."
             )
 
-    def _read_files(self, files: PathType | None) -> dict[str, Any]:
+    def _read_files(self, files: PathType | None, **kwargs: Any) -> dict[str, Any]:
```

Ruff (0.15.1)

[warning] Unused method argument: `kwargs`

(ARG002)